### PR TITLE
[Quickfix] Improve mobile conversations design

### PR DESCRIPTION
**Pull request by ghost**
screen shot 2015-09-04 at 15 51 11

### CODE REVIEW

**Comment by AugierLe42e** [comment]
Ready to review.

File: app/assets/stylesheets/bootstrap-variables.scss [hunk 1]
```diff
@@ -20,7 +20,7 @@ $brand-primary: darken(#0097FF,5%) !default; // darker creation-blue
 $brand-success: #8EDE3D !default;
 // $brand-info:            #5bc0de
 // $brand-warning:         #f0ad4e
-// $brand-danger:          #d9534f
+$brand-danger: #A80000;
```
Comment (AugierLe42e) [inline app/assets/stylesheets/bootstrap-variables.scss]: This was the original red color used both on mobile and on desktop. It was used on mobile next to BS3's default red. This is to unify the colors.

File: app/assets/stylesheets/mobile/conversations.scss [hunk 2]
```diff
@@ -1,10 +1,11 @@
+.conversations-title { margin: 0 0 20px 0; }
 .conversation {
-
   .conversation_participants {
     padding: 1rem 1.2rem;
 
     h3 {
-      margin: 0px;
+      margin: 0;
+      display: inline;
```
Comment (svbergerem) [inline app/assets/stylesheets/mobile/conversations.scss]: I guess you wanted to add `display: inline` to `.conversations-title h3` instead.

File: app/assets/stylesheets/bootstrap-variables.scss (hunk 1 above)
Comment (svbergerem) [inline app/assets/stylesheets/bootstrap-variables.scss]: **before**:
red button before

**after**:
red button after

Not sure which version I'd prefer but I tend to prefer the first one.

**Comment by AugierLe42e** [comment]
I kept the `$red` color only for the `.badge-important`.

File: app/views/notifications/index.html.haml [hunk 3]
```diff
@@ -5,12 +5,12 @@
         = t('.notifications')
       .list-group
         %a.list-group-item{class: ('active' unless params[:type] && @grouped_unread_notification_counts.has_key?(params[:type])), href: '/notifications' + (params[:show] == 'unread' ? '?show=unread' : '') }
-          %span.pull-right.badge{class: ("hidden" unless @unread_notification_count > 0)}
+          %span.pull-right.badge.badge-important{class: ("hidden" unless @unread_notification_count > 0)}
```
Comment (svbergerem) [inline app/views/notifications/index.html.haml]: I'd like to keep the gray badge in the notification view.

Comment (AugierLe42e) [inline app/views/notifications/index.html.haml]: Ok. It hough it was a BS3 regression.

**Comment by svbergerem** [comment]
Thanks!

**Pull request merged**